be/src/exec/olap_scanner.cpp

```diff
@@ -286,6 +286,8 @@ Status OlapScanner::get_batch(RuntimeState* state, RowBatch* batch, bool* eof) {
     int64_t raw_bytes_threshold = config::doris_scanner_row_bytes;
     {
         SCOPED_TIMER(_parent->_scan_timer);
+        ObjectPool tmp_object_pool;
```

Comment:
please add more comments to explain why we need this tmp object pool.

Reply:
Done.